Sources/MessagingPushAPN/MessagingPushAPN+PushConfigs.swift

```diff
@@ -2,13 +2,15 @@ import Foundation
 import UIKit
 
 extension MessagingPushAPN {
+    @available(iOSApplicationExtension, unavailable)
```

Comment:
The additions to this file match what the [FCM equivalent file](https://github.com/customerio/customerio-ios/blob/a2ccebf1dae6d891c8c699005a083a4147241130/Sources/MessagingPushFCM/MessagingPushFCM%2BPushConfigs.swift) contains.